test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -430,17 +433,15 @@ private static File createTempStore(String storePath) {
    *
    * @return major version of current JVM
    */
-  private static int getCurrentJvmMajorVersion() {
+  @VisibleForTesting
+  static int getCurrentJvmMajorVersion() {
```

Comment:
Nit: I'm kinda wondering if instead of making this visible for testing we don't just change things so that all the logic lives in a getCurrentJvmMajorVersion() impl which takes a version string as an arg:

```java
private static int getCurrentJvmMajorVersion() {
   return getJvmMajorVersion(System.getProperty("java.version"));
}

static int getJvmMajorVersion(String versionSysprop) {
   // Same logic in here (without the sysprop access)
}
```

Would probably simplify your tests quite a bit.